extensions/amp-lightbox/1.0/test-e2e/test-custom-close-button.js

```diff
@@ -45,7 +45,8 @@ describes.endtoend(
       ).to.equal(0);
     });
 
-    it('should open the lightbox', async () => {
+    // TODO(wg-components, #24195): Flaky during CI.
```

Comment:
#24195 tracks the carousel failures. There's a different issue for lightbox that can be used here: #28948.